package.json

```diff
@@ -162,7 +162,7 @@
       "brotli": false,
       "path": "./packages/teams-js/dist/esm/packages/teams-js/src/index.js",
       "import": "{ app, authentication, pages }",
-      "limit": "58.49 KB"
+      "limit": "59 KB"
```

Comment:
Hmm, in main, it is "58.59" Kb. Is there any reason why we need to increase to 59Kb?

Reply:
I think it's a bit silly to constantly be updating this to the exact 0.01kb for every change. It should be %-increase based, or at least prevent large changes (like importing a new library or an accidental import). In a prior iteration I also set it to the exact 0.01kb and it still failed because the build used the exact byte count: https://github.com/OfficeDev/microsoft-teams-library-js/actions/runs/16667552609/job/47176738074

I just set it to the nearest ceiling round KB.

Reply:
Ok, I don't have enough insights here. @sunbrk @jekloudaMSFT for awareness for the potential improvement here.